Content.Server/Clothing/Components/ClothingComponent.cs

```diff
@@ -1,113 +1,73 @@
 using Content.Server.Hands.Components;
-using Content.Server.Inventory.Components;
-using Content.Server.Items;
 using Content.Shared.Clothing;
 using Content.Shared.Interaction;
+using Content.Shared.Inventory;
 using Content.Shared.Item;
-using Content.Shared.Popups;
-using Content.Shared.Sound;
 using Robust.Shared.GameObjects;
 using Robust.Shared.GameStates;
 using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
 using Robust.Shared.Serialization.Manager.Attributes;
 using Robust.Shared.ViewVariables;
-using static Content.Shared.Inventory.EquipmentSlotDefines;
 
 namespace Content.Server.Clothing.Components
 {
     [RegisterComponent]
     [ComponentReference(typeof(SharedItemComponent))]
-    [ComponentReference(typeof(ItemComponent))]
-    [NetworkedComponent()]
+    public class ItemComponent : SharedItemComponent{}
+
+    [RegisterComponent]
+    [NetworkedComponent]
+    [ComponentReference(typeof(SharedItemComponent))]
     public class ClothingComponent : ItemComponent, IUse
```

Comment:
Not suggesting that this be done in this PR, but after this refactor how much effort would it be to remove the clothing/item component inheritance? Would it even be worth the effort, given that (AFAIK) a clothing entity is always going to be an item as well?

Reply:
ye i agree the interitance should be removed, i conciously didn't do it in this pr because it would envolve quite a bit more refactoring. more specifically, it would require *alot* of yaml changes :/

Reply:
to answer the question tho: the effort would basically mostly be in yaml